lib/blue_apron/spree_client.rb

```diff
@@ -133,6 +133,13 @@ def get_products(options = {})
       response
     end
 
+    ##
+    # Get products in a specific taxon.
+    def get_products_in_taxon(taxon_id)
+      options = { params: { id: taxon_id } }
+      get "/api/taxons/products", options
```

Comment:
Do you need to sanitize the products like in `get_products`??

Reply:
For our purposes, probably not, since our products won't have a ton of descriptions/etc. :-) But yeah, that's a good call for general correctness. Will update!